### PR TITLE
BEIS ingress v3 - Sea & Road

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] New
- [*] Update

**What category or categories does it affect?**

_ Sea Travel, Sea Freight, Road Freight, Road Travel_

**What?**

_Complete BEIS transport category emissions factors_

**Why?**

_To update the existing EFs and add any missing ones from the source._

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -3629,9 +3629,170 @@ Transport,Sea Freight,sea_freight-vessel_type_water_transport_boats_ships_ferrie
...
+Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_gt_2000_lm-tonnage_na-fuel_source_na,RoRo-ferry >2000 LM,tonne-km,not-supplied,0.050189,0.0495,0.000015,0.000674,not-supplied,not-supplied,,3,fuel_combustion,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity for roll-on/roll-off (RoRo) Ferry. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3786]:
```suggestion
Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_gt_2000_lm-tonnage_na-fuel_source_na,RoRo-ferry >2000 LM,tonne-km,not-supplied,0.050189,0.0495,0.000015,0.000674,not-supplied,not-supplied,,3,fuel_combustion,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity for roll-on/roll-off (RoRo) ferry of greater than 2000 lane meters. A lane meter is a unit of deck area in roll-on/roll-off ships.Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -3629,9 +3629,170 @@ Transport,Sea Freight,sea_freight-vessel_type_water_transport_boats_ships_ferrie
...
+Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_gt_2000_lm-tonnage_na-fuel_source_na,RoRo-ferry >2000 LM - WTT,tonne-km,not-supplied,0.011272,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity WTT for roll-on/roll-off (RoRo) Ferry. Well-to-tank (WTT) conversion factors for delivery vehicles & freighting goods should be used to report the upstream Scope 3 emissions associated with extraction/refining and transportation of the raw fuels before they are used to power the transport mode. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3787]:
```suggestion
Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_gt_2000_lm-tonnage_na-fuel_source_na,RoRo-ferry >2000 LM - WTT,tonne-km,not-supplied,0.011272,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity of WTT for roll-on/roll-off (RoRo) ferry of greater than 2000 lane meters. A lane meter is a unit of deck area in roll-on/roll-off ships. Well-to-tank (WTT) conversion factors for delivery vehicles & freighting goods should be used to report the upstream Scope 3 emissions associated with extraction/refining and transportation of the raw fuels before they are used to power the transport mode. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -3629,9 +3629,170 @@ Transport,Sea Freight,sea_freight-vessel_type_water_transport_boats_ships_ferrie
...
+Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_between_0_and_1999_lm-tonnage_na-fuel_source_na,RoRo-ferry 0 - 1999 LM,tonne-km,not-supplied,0.061139,0.0603,0.000018,0.000821,not-supplied,not-supplied,,3,fuel_combustion,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity for roll-on/roll-off (RoRo) Ferry. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3788]:
```suggestion
Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_between_0_and_1999_lm-tonnage_na-fuel_source_na,RoRo-ferry 0 - 1999 LM,tonne-km,not-supplied,0.061139,0.0603,0.000018,0.000821,not-supplied,not-supplied,,3,fuel_combustion,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity for roll-on/roll-off (RoRo) ferry of between 0 and 1999 lane meters. A lane meter is a unit of deck area in roll-on/roll-off ships. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -3629,9 +3629,170 @@ Transport,Sea Freight,sea_freight-vessel_type_water_transport_boats_ships_ferrie
...
+Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_between_0_and_1999_lm-tonnage_na-fuel_source_na,RoRo-ferry 0 - 1999 LM - WTT,tonne-km,not-supplied,0.013731,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity WTT for roll-on/roll-off (RoRo) Ferry. Well-to-tank (WTT) conversion factors for delivery vehicles & freighting goods should be used to report the upstream Scope 3 emissions associated with extraction/refining and transportation of the raw fuels before they are used to power the transport mode. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:3789]:
```suggestion
Transport,Sea Freight,sea_freight-vessel_type_roro_ferry-route_type_na-vessel_length_between_0_and_1999_lm-tonnage_na-fuel_source_na,RoRo-ferry 0 - 1999 LM - WTT,tonne-km,not-supplied,0.013731,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,GB,,2022/03/20,Emission intensity of WTT for roll-on/roll-off (RoRo) ferry of between 0 and 1999 lane meters. A lane meter is a unit of deck area in roll-on/roll-off ships. Well-to-tank (WTT) conversion factors for delivery vehicles & freighting goods should be used to report the upstream Scope 3 emissions associated with extraction/refining and transportation of the raw fuels before they are used to power the transport mode. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```

**Review by patstirling — CHANGES_REQUESTED** [review]

**Review by AnastasiaLobanova — DISMISSED** [review]
Looks good but there are some dates issues - 2022/3/20 and 2022/03/20 in the Road Freight and Road Travel part. I remember it was before also not sure if we have decided to format dates afterwards.

And just a clarification - year calculated is missing because  BEIS provides no info on this?

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**